src/main.c

```diff
@@ -2720,7 +2761,7 @@ static void *run_loop(void *_thread_index)
     h2o_multithread_register_receiver(conf.threads[thread_index].ctx.queue, &conf.threads[thread_index].memcached,
                                       h2o_memcached_receiver);
 
-    if (conf.thread_map.entries[thread_index] >= 0) {
+	if (conf.thread_map.entries[thread_index] >= 0) {
```

Comment:
I believe this is an unintentional change?

Reply:
yes it is unintentional - sorry about that.